mixins/visual/responsive-visual.vue

```diff
@@ -19,6 +19,29 @@ export default
 		portraitVideo: Object
 	}
 
+	head: ->
+		return {} unless @isResponsiveImage and @preload
+
+		return link: [
+			{
+				rel: 'preload'
+				as: 'image'
+				imagesrcset: @makeSrcset @landscape.props.image
+				href: @landscape?.props?.image?.url
+				imagesizes: @sizes
```

Comment:
Prevent an attribute value of `undefined` from being rendered when this is empty

```suggestion
				imagesizes: @sizes || ''
```